install/deploy_final_non-compute_node.md

```diff
@@ -353,7 +357,7 @@ The steps in this section load hand-off data before a later procedure reboots th
     exit
     exit
     # typescript exited
-    rsync -rltDv -P /metal/bootstrap ncn-m001:/metal/ && rm -rfv /metal/bootstrap
+    rsync -rltDv -P /metal/bootstrap ncn-m002:/metal/ && rm -rfv /metal/bootstrap
```

Comment:
This rsync command is being run on ncn-m002. This change confuses me. It doesn't look like you included this portion of the procedure in your testing. How were you intending this change to work?

Reply:
The original commands in the rsync in the doc as written was being executed/run from ncn-m001, but it was pointing to m001 in a circular reference. If its run from m001 it needs to copy to ncn-m002.

Reply:
The original commands in the doc have you run exit first, which logs you out of ncn-m001 and back to ncn-m002. The second exit exits the typescript. Then the rsync is run.

This was discussed in an earlier comment I made that you implemented: https://github.com/Cray-HPE/docs-csm/pull/2275#discussion_r957474134